exosphere-shared/src/docker-helper.ls

```diff
@@ -9,6 +9,12 @@ class DockerHelper
     child_process.exec-sync('docker ps -a --format {{.Names}}', 'utf8') |> (.includes container)
 
 
+  @ensure-container-is-running = (container, image) ->
+    return if child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}' , 'utf8') |> (.includes "#{container}/Up")
+    @remove-container that if @container-exists container
```

Comment:
From the Livescript documentation, "`that` refers implicitly to the value of the condition. It will unwrap existence checks."... I was using it incorrectly here. 

I changed this line to restart the container in this case anyway. The first line returns if the container is running, the second restarts the container if it exists but has been stopped, and the third runs the container if it does not exist at all.